lld/ELF/LinkerScript.h

```diff
@@ -86,9 +86,10 @@ struct SectionCommand {
 
 // This represents ". = <expr>" or "<symbol> = <expr>".
 struct SymbolAssignment : SectionCommand {
-  SymbolAssignment(StringRef name, Expr e, std::string loc)
+  SymbolAssignment(StringRef name, Expr e, std::string loc,
+                   unsigned symOrder = 0)
```

Comment:
There are only a couple more calls to the constructor in the same file. Would it be better to make the index explicit at the call site?

Not a strong opinion, and happy for it to be left as is.